src/mmw/js/src/data_catalog/templates/resultDetailsCinergi.html

```diff
@@ -4,27 +4,111 @@
             <i class="fa fa-arrow-left black"></i> Back
         </button>
         <h2>
-            {{ title }}
+            Resource: {{ title }}
         </h2>
+        <table class="table-data-detail">
+            <tr>
+                <th>Contacts</th>
+                <td>
+                    {% for contact in contact_people %}
+                         {{ contact }}<br />
+                    {% endfor %}
+                </td>
+            </tr>
+            <tr>
+            <tr>
+                <th>Organizations</th>
+                <td>
+                    {% for org in contact_organizations %}
+                         {{ org }}<br />
```

Comment:
Maybe instead of a `<br />` we should wrap these in a `<p>` so that it is easier to tell when a long name is wrapping vs a new name starting:

![image](https://user-images.githubusercontent.com/1430060/32064058-cf1cc17c-ba46-11e7-9ad6-e3fa33a98e91.png)